### PR TITLE
Improved lobby scenery

### PR DESCRIPTION
**Pull request by jngo102**
- Random spawning of scenery objects
- Implemented object pooling in a class

### CODE REVIEW

File: src/assets/autoload/helpers.gd [hunk 1]
```diff
@@ -1,5 +1,12 @@
 extends Node
 
+func choose(array: Array):
```
Comment (TheSecondReal0) [inline src/assets/autoload/helpers.gd]: Maybe change this function name to ```pick_random()```?

**Review by TheSecondReal0 — CHANGES_REQUESTED** [review]
There's one small change to a function name I'd like you to make but other than that everything looks good! The code looks good and it works perfectly!

File: src/assets/autoload/helpers.gd (hunk 1 above)
Comment (jngo102) [reply src/assets/autoload/helpers.gd]: Changed

**Review by TheSecondReal0 — APPROVED** [review]
Code is solid, works perfectly, ready to merge!

**Pull request merged**